src/js/pages/system/OverviewDetailTab.js

```diff
@@ -22,8 +23,11 @@ import Loader from "../../components/Loader";
 import MarathonStore
   from "../../../../plugins/services/src/js/stores/MarathonStore";
 import MetadataStore from "../../stores/MetadataStore";
+import MesosStateStore from "../../stores/MesosStateStore";
 import Page from "../../components/Page";
 import VersionsModal from "../../components/modals/VersionsModal";
+import { msToRelativeTime } from "../../utils/DateUtil";
```

Comment:
`msToRelativeTime` operates on milliseconds but we get a unix timestamp in _seconds_ from the API thus you see 47 ago on the screen